documentation/modules/ROOT/pages/connectors/mysql.adoc

```diff
@@ -6,20 +9,2563 @@
 :context: debezium
 
 [id="debezium-connector-for-mysql"]
-= {prodname} Connector for MySQL
+= {prodname} connector for MySQL
 
-MySQL has a binary log (binlog) that records all operations in the order in which they are committed to the database. This includes changes to table schemas and the data within tables. MySQL uses the binlog for replication and recovery.
+MySQL has a binary log (binlog) that records all operations in the order in which they are committed to the database. This includes changes to table schemas as well as changes to the data in tables. MySQL uses the binlog for replication and recovery.
 
-The {prodname} MySQL connector reads the binlog and produces change events for row-level `INSERT`, `UPDATE`, and `DELETE` operations and records the change events in a Kafka topic. Client applications read those Kafka topics.
+The {prodname} MySQL connector reads the binlog, produces change events for row-level `INSERT`, `UPDATE`, and `DELETE` operations, and emits the change events to Kafka topics. Client applications read those Kafka topics.
 
 As MySQL is typically set up to purge binlogs after a specified period of time, the MySQL connector performs an initial _consistent snapshot_ of each of your databases. The MySQL connector reads the binlog from the point at which the snapshot was made.
 
-The following sections provide more detailed information on how the {prodname} MySQL connector functions, instructs you on how to set it up, as well as deploy and troubleshoot the connector.
+ifdef::product[]
+Information and procedures for using a {prodname} MySQL connector are organized as follows:
+
+* xref:how-debezium-mysql-connectors-work[]
+* xref:descriptions-of-debezium-mysql-connector-data-change-events[]
+* xref:how-debezium-mysql-connectors-map-data-types[]
+* xref:setting-up-mysql-to-run-a-debezium-connector[]
+* xref:deploying-debezium-mysql-connectors[]
+* xref:monitoring-debezium-mysql-connector-performance[]
+* xref:how-debezium-mysql-connectors-handle-faults-and-problems[]
+
+endif::product[]
+
+// Type: assembly
+// ModuleID: how-debezium-mysql-connectors-work
+// Title: How {prodname} MySQL connectors work
+[[how-the-mysql-connector-works]]
+== How the connector works
+
+To optimally configure and run a {prodname} MySQL connector, it is helpful to understand how the connector tracks the structure of tables, exposes schema changes, performs snapshots, and determines Kafka topic names. An overview of the MySQL topologies that the connector supports is useful for planning your application. 
```

Comment:
I moved that last sentence in the introductory paragraph, about topologies, to be the first sentence. 
I moved the toplogies section to be the first subsection in How the connector works. 
I don't think the list of xref links is needed in the text panel because they will appear in the navigation panel on the right upstream.  Downstream, you see the links for the subsections only if you clicked the main section that contains them, and that's why I add them for downstream rendering. In other words, downstream, if you are scrolling content, the navigation panel on the left does not necessarily show you what you are looking at, and it is very long because it is for the whole book. Let me know if you want something different. Otherwise, this update will be in the next commit.